test/NuGet.Core.FuncTests/Dotnet.Integration.Test/SignCommandTestFixture.cs

```diff
@@ -0,0 +1,381 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+using NuGet.Packaging.Signing;
+using NuGet.Test.Utility;
+using Test.Utility.Signing;
+
+namespace Dotnet.Integration.Test
+{
+    /// <summary>
+    /// Used to bootstrap functional tests for signing.
+    /// </summary>
+    public class SignCommandTestFixture : IDisposable
```

Comment:
I remember copying code from `NuGet.CommandLine.FuncTest/Commands/SignCommandTestFixture.cs` when I was working on the `dotnet sign` command. Can we move this class file to `TestUtility` so that the implementation can be shared in `NuGet.CommandLine.FuncTest` & `Dotnet.Integration.Test` projects without creating a new class?

https://github.com/NuGet/NuGet.Client/blob/dev/test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/Commands/SignCommandTestFixture.cs

Reply:
Thanks @kartheekp-ms. Yes, at the beginning, the two fixtures are the same. 
But later on, I made some change in the test/NuGet.Core.FuncTests/Dotnet.Integration.Test/SignCommandTestFixture.cs, including:
1).Change the signing test fixture a little bit, to set up a 3 level certs to sign, as on Mac, a self-issued trusted cert needs to use many commands to add to trusted store, according to dotnet/runtime#49528 (comment)
2). Remove NuGetExePath as the dotnet integration tests doesn't need nuget.exe.
So the two are different now.
And I also found the test fixture usually stays with the test class in our repo. So I guess it's okey to not merge the two fixtures into test.utility?

Reply:
Thanks for pointing out that fixture could be shared between assemblies. Created a tracking issue https://github.com/NuGet/Client.Engineering/issues/950 to consolidate the two `SignCommandTestFixture.cs`.